<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_window</class>
 <widget class="QDialog" name="Home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>753</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFFFFF</string>
  </property>
  <widget class="QFrame" name="Big_frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>101</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #4C0099; border: 1px ; border-radius: 20px; </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Account_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Acc</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Загрузки/Картинки/png-transparent-computer-icons-google-account-user-email-miscellaneous-rim-area.png</normaloff>../Загрузки/Картинки/png-transparent-computer-icons-google-account-user-email-miscellaneous-rim-area.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>19</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Transactions_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Transa</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Cards_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue;color: white;</string>
    </property>
    <property name="text">
     <string>Card</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>971</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Acc_page">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>891</width>
       <height>581</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>581</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>561</width>
         <height>541</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:rgba(211, 211, 211, 0.5);
border-radius: 20px;
}

QLabel{
color: black;
}
QLineEdit{
background-color: white;
border-radius: 6px;
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>53</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>124</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>190</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>252</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>374</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>424</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Surname_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="Name_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>120</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="Middle_name_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>190</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="Birthday_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>250</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="Login_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>370</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="Email_line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>420</y>
          <width>249</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Card_page">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>611</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView{
background-color: rgba(211, 211, 211, 0.5);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Right_swap_button">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>200</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Left_swap_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="menu_card_frame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>881</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color:lightgray;
border-radius: 15px;
}

QPushButton{
border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="Card_data_button">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Данные карты</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Transfer_money_button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Перевести средства</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Register_new_card_button">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Зарегестрировать карту</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_change_size_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:17px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Pay_the_bill_button">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Оплатить счёт</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Transactions_page">
    <widget class="QTableWidget" name="Transactions_table_widget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Состояние</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>501</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color:rgba(211, 211, 211, 0.5);
border-radius: 20px;
}

QLabel{
color: black;
}
QLineEdit{
background-color: white;
border-radius: 6px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>34</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Номер карты</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Card_number_line">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
